### ISSUE TITLE
Update QT to 5.9.7

### ISSUE THREAD

**Issue by cevap** [cevap/ion#66]
e4354b162b4ab481f9274b3c0e0b7706ee4c6685 - Update QT to 5.9.7, compilation for linux breaks

**Comment by cevap** [comment]
Builds for macos build, but there is a bug with pink background

**Comment by cevap** [comment]
Builds for windows fail with following error:

```
qwin10helpers.cpp:60:37: fatal error: uiviewsettingsinterop.h: No such file or directory
 #  include <uiviewsettingsinterop.h>
```

full lxc.config, installation and build logs are attached:
[build.log](https://github.com/cevap/ion/files/2683028/build.log)
[install.log](https://github.com/cevap/ion/files/2683029/install.log)

**Comment by cevap** [comment]
fixed and closed in #105
